contrib/debian/control

```diff
@@ -3,7 +3,7 @@ Section: unknown
 Priority: optional
 Maintainer: Matthias Dellweg <dellweg@atix.de>
 #Build-Depends: debhelper (>= 9), dh-python, libglib2.0-dev, libdbus-glib-1-dev, libgtk2.0-dev, libglib2.0-dev, libgconf2-dev, libnotify-dev, lsb-release, pkg-config, python-all, python-setuptools, python3-all, python3-setuptools, intltool, python-dbus, python3-dbus, python-kitchen, python3-kitchen, python-flake8, python3-flake8, python-freezegun, python3-freezegun, python-polib, python3-polib, python-coverage, python3-coverage, python-dev, python3-dev, libssl-dev
-Build-Depends: debhelper (>= 9), dh-python, libglib2.0-dev, libdbus-glib-1-dev, libgtk2.0-dev, libglib2.0-dev, libgconf2-dev, libnotify-dev, lsb-release, pkg-config, python-all, python-setuptools, intltool, python-dbus, python-kitchen, python-flake8, python-freezegun, python-polib, python-coverage, python-dev, libssl-dev
+Build-Depends: debhelper (>= 9), dh-python, libglib2.0-dev, libdbus-glib-1-dev, libgtk2.0-dev, libglib2.0-dev, libgconf2-dev, libnotify-dev, lsb-release, pkg-config, python-all, python-setuptools, intltool, python-dbus, python-kitchen, python-flake8, python-freezegun, python-polib, python-coverage, python-dev, libssl-dev, dpkg-dev
```

Comment:
`dpkg-dev` is part of `build-essential`, so you don't need to pull it (unless you need a specific version of it, which is not the case here)